tests/test_scheduler_dashboard.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+from tempfile import TemporaryDirectory
+from pathlib import Path
+import bokeh.plotting
+import bokeh.io
+
+from rubin_sim.scheduler.model_observatory import ModelObservatory
+from rubin_sim.scheduler.example import example_scheduler
+
+from schedview.app.scheduler_dashboard.scheduler_dashboard import (Scheduler, scheduler_app)
+
+# Schedview methods
+from schedview.collect.scheduler_pickle import read_scheduler
+from schedview.compute.scheduler import make_scheduler_summary_df
+from schedview.compute.survey import make_survey_reward_df, compute_maps
+
+# Objects to test instances against
+from rubin_sim.scheduler.schedulers.core_scheduler import CoreScheduler
+from rubin_sim.scheduler.features.conditions import Conditions
+from panel.widgets import Tabulator
+from collections import OrderedDict
+import pandas as pd
+
+from astropy.time import Time
+from pandas import Timestamp
+from zoneinfo import ZoneInfo
+from datetime import datetime
+
+"""
+Tests I usually perform:
+
+    1. Load valid pickle.
+    2. Choose date.
+    3. Choose tier.
+    4. Select survey.
+    5. Select basis function (array).
+    6. Select basis function (finite scalar).
+    7. Select basis function (-inf scalar).
+    8. Choose survey map (basis function).
+    9. Choose survey map (sky brightness).
+   10. Choose nside.
+   11. Choose color palette.
+   12. Check tooltip reflects selection data.
+   13. Choose invalid date.
+   14. Choose invalid pickle.
+   15. Load two pickles, one after the other.
+
+
+Notes
+-----
+
+    - Unit tests are not supposed to rely on each other.
+    - Unit tests are run in alphabetical order.
+
+"""
+
+TEST_PICKLE = "/Users/me/Documents/2023/ADACS/Panel_scheduler/Rubin_scheduler_dashboard/example_pickle_scheduler.p.xz"
+TEST_DATE = datetime(2023, 8, 1, 23, 0)
+DEFAULT_TIMEZONE = 'America/Santiago'
+
+
+class test_scheduler_dashboard(unittest.TestCase):
```

Comment:
Class names start with upper case letters and doesn't use `_`. This should be `TestSchedulerDashboard`